src/Navigation.js

```diff
@@ -127,7 +129,8 @@ const EventsStack = createStackNavigator(
     [EVENT_LIST]: { screen: withShadow(EventsScreen) },
     [EVENT_DETAILS]: { screen: EventDetailsScreen },
     [EVENT_CATEGORIES_FILTER]: { screen: CategoriesFilterScreen },
-    [FILTER_MODAL]: { screen: FilterScreen }
+    [FILTER_MODAL]: { screen: FilterScreen },
+    [DATE_FILTER_MODAL]: { screen: DateFilterScreen }
```

Comment:
Can we not call it modal, please? It's just a filter screen. The original was a "filter modal" because it was an actual modal at first (popping up from the bottom), but then we changed it. It should just be `DATE_FILTER` IMO.

Reply:
`EVENT_DATE_FILTER`?
Then maybe `FILTER_MODAL` -> `EVENT_ATTRIBUTE_FILTER` or similar

Reply:
done